src/angular/8-state-city-options/state-city-options.md

```diff
@@ -109,7 +109,7 @@ export class UserComponent {
 }
 ```
 
-The syntax above is equivalent to the code below:
+The syntax above is equivalueent to the code below:
```

Comment:
@chasenlehara `equivalueent`? I think you replaced every val with value and this one not intended 🤣